pkg/types/installconfig.go

```diff
@@ -30,6 +30,7 @@ var (
 	PlatformNames = []string{
 		aws.Name,
 		azure.Name,
+		baremetal.Name,
```

Comment:
I think we're going to need to refactor this a bit. We want baremetal to show up as an option only when openshift-baremetal-installer gets built with the +baremetal tag.

We'll need to register it dynamically into PlatformNames, kind of like how we do https://github.com/openshift/installer/blob/master/pkg/types/baremetal/validation/libvirt.go.

@abhinavdahiya Would that be ok for you?

Reply:
Can't we just replicate what libvirt is doing now? Here: https://github.com/openshift/installer/blob/master/pkg/types/installconfig_libvirt.go

Reply:
Yea that's exactly what I was suggesting, I didn't notice another platform had already done that.

IMHO we should create installconfig_baremetal.go since we have our own tag.

Reply:
Awesome. Already did that in the new commit.